src/sage.h

```diff
@@ -14,7 +14,7 @@ extern "C" {
 
     /* API for sage */
     extern int run_sage(const int ThisTask, const int NTasks, const char *param_file, void **params);
-    extern int finalize_sage(void *run_params);
+    extern int finalize_sage(void *params);
```

Comment:
This seems like an odd change but I have made it so that `run_params` is always of type `struct params` whereas `params` is of type `void`. 

(Noting for current me since past me did not explain this change)